Dockerfile

```diff
@@ -17,20 +17,21 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
-
-FROM adoptopenjdk/openjdk11:alpine-slim AS build
+FROM eclipse-temurin:11-jdk-focal AS build
```

Comment:
Switching to focal seems to open a Pandora's box of Python questions as Python2 is no more the default.
Has anyone started looking into that already, maybe @sschuberth  @mnonnenmacher ?

Reply:
@heliocastro did touch this topic in https://github.com/oss-review-toolkit/ort/pull/3902 (however, that's doing too many things at once for my taste). Maybe it's easier if we tackle the ScanCode upgrade first?

Reply:
Thanks @sschuberth , I'll close this draft then. Upgrading ScanCode first seems very reasonable.
Do we have plans to drop Python2 support from ORT after the ScanCode upgrade @oss-review-toolkit/core-devs  (so that switching to focal would become simpler)?

Reply:
> Do we have plans to drop Python2 support from ORT after the ScanCode upgrade

I don't think we can (fully drop Python 2 from the Docker image), as we'd still need to be able to analyze Python 2 projects, or?

Reply:
From the complexity the Python2 support adds it would be great to get rid of it. However, if it's a requirement then we have to keep it. Would it make sense to discuss this in the dev meeting ?

Reply:
Feel free to put it onto the agenda. However, I'm hoping we don't get "from the hundredth to the thousandth" in that ever-lasting Docker-ScanCode-Python vicious circle.